src/zjs_ocf_ble.c

```diff
@@ -59,6 +59,7 @@ jerry_value_t zjs_ocf_set_ble_address(const jerry_value_t function_val,
                                       const jerry_value_t argv[],
                                       const jerry_length_t argc)
 {
+#ifndef ZJS_CONFIG_BLE_ADDRESS
```

Comment:
I'm still not sure whether this makes sense or the JS should override command line. But document this behavior in the setBleAddress doc I guess, if it isn't already.